packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -505,6 +509,12 @@ function Preview({
           isKeydown && setAnchorPoint({ x: 0.5, y: 0.5 });
           setIsMultiTouching(isKeydown);
         }
+
+        if (isMultitouchKeyPressed && !isKeydown) {
+          sendMultiTouch(touchPoint, "Up");
+          setIsPressing(false);
+        }
```

Comment:
I'm not sure if I'm getting the shift/alt part. Next if takes care of shift part those two test if multitouch should be activated or not. 

I didn't want to merge to avoid nesting ifs, as imo those can be harder to follow. But I split it even more in https://github.com/software-mansion/radon-ide/pull/716. Where we basically get `if (start multitouch)` and `if (finish multitouch)`, does it look better?

I definitly agree with confusing name, although I would update it in https://github.com/software-mansion/radon-ide/pull/716.